lib/delayed_paperclip/jobs/delayed_job.rb

```diff
@@ -8,6 +9,11 @@ class DelayedJob < Struct.new(:instance_klass, :instance_id, :attachment_name)
       if Gem.loaded_specs['delayed_job'].version >= Gem::Version.new("2.1.0")
 
         def self.enqueue_delayed_paperclip(instance_klass, instance_id, attachment_name)
+          ActiveSupport::Deprecation.warn(<<-MESSAGE)
```

Comment:
@morgoth did you have a chance to double check whether this shows up in the main server log and not just background job logs?